CardsCashCasino/Data/UserHand.cs

```diff
@@ -1,12 +1,12 @@
 ﻿/*
  *  Module Name: UserHand.cs
- *  Purpose: Models the user's hand of cards.
+ *  Purpose: Models the user's hand of cards. This class is used in the Blackjack game.
```

Comment:
Remove this comment. UserHand should also be used in Poker.

The idea is that we only ever have one "UserHand" class that is communicated with by ALL three games. That might require some refactoring of the methods, but its intended to keep the project relatively organized.